src/App.php

```diff
@@ -2006,4 +2006,21 @@ public function internalRedirect($toUrl = '', $ssl = false)
 		$redirectTo = $this->getBaseURL($ssl) . '/' . ltrim($toUrl, '/');
 		Core\System::externalRedirect($redirectTo);
 	}
+
+	/**
+	 * Redirects to another URL in case
+	 * Should only be used if it isn't clear if the URL is either internal or external
+	 *
+	 * @param string $toUrl The target URL
+	 *
+	 */
+	public function redirect($toUrl)
+	{
+		if (filter_var($toUrl, FILTER_VALIDATE_URL))
```

Comment:
Code standards: Please move the opening curly brace on the same line as the `if`.